nx/include/switch/services/ncm_types.h

```diff
@@ -130,12 +134,30 @@ typedef struct {
     u8 reserved[0x8];            ///< Unused.
 } NcmPatchMetaExtendedHeader;
 
-/// AddOnContentMetaExtendedHeader
+/// AddOnContentMetaExtendedHeader [15.0.0+]
+typedef struct {
+    u64 application_id;                  ///< ApplicationId of this add-on-content's corresponding application.
+    u32 required_application_version;    ///< Version of the application required by this add-on-content.
+    u8 content_accessibility_individual; ///< ContentAccessibilityIndividual.
```

Comment:
This should just be content_accessibilities, "content_accessibility_individual" is incorrect.

Reply:
Gotta update Switchbrew, then. Thanks.